files/galaxy/config/job_metrics_conf.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
this is not needed. Look in all.yml for job metrics conf definition that works with the galaxyproject.galaxy role